garminexport/garminbackup.py

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env python
+"""Performs (incremental) backups of activities for a given Garmin Connect
+account.
+The activities are stored in a local directory on the user's computer.
+The backups are incremental, meaning that only activities that aren't already
+stored in the backup directory will be downloaded.
+"""
+import argparse
+from datetime import timedelta
+import getpass
+from garminexport.garminclient import GarminClient
+import garminexport.backup
+from garminexport.backup import export_formats
+from garminexport.retryer import (
+    Retryer, ExponentialBackoffDelayStrategy, MaxRetriesStopStrategy)
+import logging
+import os
+import re
+import sys
+import traceback
+
+logging.basicConfig(
```

Comment:
If this is to be used as a library, I'd like the library client to be able to determine logging behavior. Therefore, move all log-related setup out to the "main" script.

Reply:
dealing with logging is not my strongest. I tried to rearrange logging related lines to the best of my understanding. for example, there is a new `garminexport/logging_config.py` that stores `LOG_LEVELS` dictionary. would appreciate some more specific advice if this change is not what you expect.